docs/network-flow-visibility.md

```diff
@@ -550,7 +550,14 @@ kubectl logs <ipfix-collector-pod-name> -n ipfix
 
 ### Grafana Flow Collector
 
-Grafana Flow Collector feature is only available for releases starting from Antrea v1.6.
+The Grafana Flow Collector was added in Antrea v1.6.0. In Antrea v1.7.0, we
+start to move the network observability and analytics functionalities of Antrea
+to [Project Theia](https://github.com/antrea-io/theia), including the Grafana
+Flow Collector. Going forward, further development of the Grafana Flow Collector
+will be in the Theia repo, but we will keep the existing implementation in the
+Antrea main repo for a while before deprecating it. This section talks about the
```

Comment:
I'm not sure whether we need this sentence or not - "This section talks about the Grafana Flow Collector in the Antrea repo." It's not very clear to me what are the differences between "Grafana Flow Collector in the Antrea repo" and "the Theia Grafana Flow Collector".

Reply:
Understood. Yes, they will differ in the future releases. 

I feel it's a bit of hard to tell these differences by looking at the phrases - "This section talks about the Grafana Flow Collector in the Antrea repo. For the Theia Grafana Flow Collector and other Theia features ... "  How about removing the first sentence, and change the second one into "To try the most updated Grafana Flow Collector and other Theia features...". 

No strong opinion, just feel slightly confused when reading at these two sentences at my first glance.

Reply:
I tried to rephrase. Could you check?